Dockerfile

```diff
@@ -1,3 +1,15 @@
+FROM --platform=$BUILDPLATFORM node:21.6-alpine3.18 AS client-builder
+WORKDIR /ui
+# cache packages in layer
+COPY ui/package.json /ui/package.json
+COPY ui/package-lock.json /ui/package-lock.json
+RUN --mount=type=cache,target=/usr/src/app/.npm \
+    npm set cache /usr/src/app/.npm && \
+    npm ci
+# install
+COPY ui /ui
+RUN npm run build
+
 FROM node:18-alpine
 LABEL org.opencontainers.image.title="MindsDB" \
     org.opencontainers.image.description="Streamline AI development with MindsDB in your Docker environment. Deploy, manage, and scale your AI models seamlessly." \
```

Comment:
in the detailed-description make sure we provide same information as on the website, or copy it from GitHub readme

Reply:
Done. I have updated the labels with some information from our main repository's README. Please let me know if this is OK.